message_ix/model/MESSAGE/model_core.gms

```diff
@@ -946,11 +976,34 @@ ACTIVITY_BOUND_UP(node,tec,year,mode,time)$( map_tec_act(node,tec,year,mode,time
 * unless explicitly stated otherwise.
 ***
 ACTIVITY_BOUND_LO(node,tec,year,mode,time)$( map_tec_act(node,tec,year,mode,time) )..
-    SUM(vintage$( map_tec_lifetime(node,tec,vintage,year) ), ACT(node,tec,vintage,year,mode,time) ) =G=
+    SUM(
+	vintage$( map_tec_lifetime(node,tec,vintage,year) ),
+	ACT(node,tec,vintage,year,mode,time)
+    )
+    =G=
     bound_activity_lo(node,tec,year,mode,time)
 %SLACK_ACT_BOUND_LO% - SLACK_ACT_BOUND_LO(node,tec,year,mode,time)
 ;
 
+***
+* Equation ACTIVITY_BOUND_ALL_MODES_LO
+* """"""""""""""""""""""""""""""""""""
+* This constraint provides lower bounds of a technology activity across all modes and vintages.
+*
```

Comment:
Please add "This constraint is only added if the lower bound across all modes is different from zero."